### DISCUSSION TITLE
Motion Watchdog timeout

### DISCUSSION THREAD

**Discussion by bad-friday** [Motion-Project/motion#1730] [General]
Always get «motion_watchdog: Thread 1 - Watchdog timeout did NOT restart, killing it» after a few minutes

Pi5 4GB, PI-HQ-Camera, Bookworm, motion 4.6.0 build from source on 15th December 2023.
[log.txt](https://github.com/Motion-Project/motion/files/13702645/log.txt)

**Comment by tosiara** [comment]
How do you run motion?
Can you check if any solution from this thread works for you?
https://github.com/Motion-Project/motion/discussions/1722#discussioncomment-7880032

**Reply by bad-friday** [reply]
As far as I can see they are talking about webcams and not on Bookworm.

**Reply by tosiara** [reply]
the comment I linked specifically mentions bookworm and libcamerify

**Comment by bad-friday** [comment]
Apparently I was able to solve the problem. The resolution is now 1920x1440. I changed the video device to video1, mmalcam_name to vc.ril.camera, movie_quality to 0, movie_bps to 4000000. Motion has been running for more than an hour now.
Edit:
After 2.5 hours it happened again

**Comment by bad-friday** [comment]
I think I'll give up and put the Pi 4 with Bullseye back into operation. 
I've already spent days trying to get Motion running under Bookworm.

**Comment by bad-friday** [comment]
But now I'm angry. 
The Computer from Raspberry, HQ camera from Raspberry, OS from Rapberry. But Motion doesn't work with it.
Configaration unchanged Raspberry-V3 Camera connected Motion started, which has now been running for more than three hours without an interruption.
